emily/lambda/src/config.rs

```diff
@@ -0,0 +1,36 @@
+use aws_sdk_dynamodb::Client;
+
+/// Temporary chainstate Table name.
+pub static CHAINSTATE_TABLE_NAME: &str = "ChainstateTable-xxxxxxxxxxxx-us-west-2-local";
+
+/// Temporary withdrawal Table name.
+pub static WITHDRAWAL_TABLE_NAME: &str = "WithdrawalTable-xxxxxxxxxxxx-us-west-2-local";
+
+/// Temporary deposit Table name.
+pub static DEPOSIT_TABLE_NAME: &str = "DepositTable-xxxxxxxxxxxx-us-west-2-local";
+
+/// Emily lambda settings.
+#[derive(Debug)]
+pub struct Settings {
+    /// Whether the Emily lambda is running locally.
+    pub is_local: bool,
+    /// Deposit table name.
+    pub deposit_table_name: String,
+    /// Withdrawal table name.
+    pub withdrawal_table_name: String,
+    /// Chainstate table name.
+    pub chainstate_table_name: String,
+}
+
+/// Lambda Context
+pub struct LambdaContext {
+    /// Lambda settings.
+    pub settings: Settings,
+    /// DynamoDB Client.
+    pub dynamodb_client: Client,
+}
+
+// TODO:
```

Comment:
I normally agree, but I'm curious if your opinion changes when its for nits like this. something like this for example is not necessary but is nice to have done; I think that could possibly fall into "untracked work that someone can do if they see it" because there is an overhead risk to too many issues.